resources/eventing/charts/nats/values.yaml

```diff
@@ -171,22 +271,48 @@ statefulSetPodLabels:
 # Annotations to add to the NATS Service
 serviceAnnotations: {}
 
+# additionalContainers are the sidecar containers to add to the NATS StatefulSet
+additionalContainers: []
+
+# additionalVolumes are the additional volumes to add to the NATS StatefulSet
+additionalVolumes: []
+
+# additionalVolumeMounts are the additional volume mounts to add to the nats-server and nats-server-config-reloader containers
+additionalVolumeMounts: []
+
 cluster:
   enabled: false
   name: eventing-nats
   replicas: 3
   noAdvertise: false
 
+  # Explicitly set routes for clustering.
+  # When JetStream is enabled, the serverName must be unique in the cluster.
+  extraRoutes: []
+
+  # authorization:
+  #   user: foo
+  #   password: pwd
+  #   timeout: 0.5
+
 appProtocol:
   enabled: true
 
 # Cluster Domain configured on the kubelets
 # https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
 k8sClusterDomain: cluster.local
 
+# Define if NATS is using FQDN name for clustering (i.e. nats-0.nats.default.svc.cluster.local) or short name (i.e. nats-0.nats.default).
+useFQDN: true
+
 # Add labels to all the deployed resources
 commonLabels: {}
 
+
+# podManagementPolicy controls how pods are created during initial scale up,
+# when replacing pods on nodes, or when scaling down.
+podManagementPolicy: Parallel
```

Comment:
Is `Parallel` better for us or `OrderedReady`?

Reply:
It is a good question, let's discuss with the team.

Reply:
let's discuss that with the team

Reply:
After a discussion with the team, we decided to go with `Parallel` option